unit_test/sparse/Test_Sparse_spmv.hpp

```diff
@@ -10,6 +10,7 @@
 
 #include "KokkosKernels_Controls.hpp"
 #include "KokkosKernels_default_types.hpp"
+#include "Cuda/Kokkos_Cuda_Half.hpp"
```

Comment:
I would have liked a separate file for bsr spmv. This compilation unit will soon be the bottleneck